### PR TITLE
Added installation guide for Mattermost.

### PR DESCRIPTION
**Pull request by mugdha-adhav**
Adding subsection in Installation for Slack and Mattermost.

### CODE REVIEW

**Comment by ssudake21** [comment]
@mugdha-adhav  I just looked at documentation . Are we having same copy at Installation section and slack page as well ?

**Comment by mugdha-adhav** [comment]
> @mugdha-adhav I just looked at documentation . Are we having same copy at Installation section and slack page as well ?

Installation section explains steps to install botkube on k8s cluster. Whereas Slack & Mattermost has (specific installation steps + install on k8s with config values)

**Comment by PrasadG193** [comment]
Updated:
- Configuration page with Mattermost and ELS fields
- Added new feature `Support for multiple interfaces` in the features list

File: content/configuration/_index.md [hunk 1]
```diff
@@ -41,21 +41,47 @@ The configuration file contains,
...
       token: 'SLACK_API_TOKEN'
-
+  
+    # Settings for Mattermost
+    mattermost:
+      enabled: false
+      url: 'MATTERMOST_SERVER_URL'                # e.g https://example.com:9243
+      token: 'MATTERMOST_TOKEN'
+      team: 'MATTERMOST_TEAM'
+      channel: 'MATTERMOST_CHANNEL'
```
Comment (PrasadG193) [inline content/configuration/_index.md]: @mugdha-adhav can you please add one liner description for these fields? Like we have for `resources`

Comment (mugdha-adhav) [reply content/configuration/_index.md]: Made required changes.

**Pull request merged**